src/neuroconv/datainterfaces/ecephys/axona/axonadatainterface.py

```diff
@@ -11,11 +9,9 @@
 from ....utils import get_schema_from_method_signature, FilePathType
 
 
-class AxonaRecordingExtractorInterface(BaseRecordingExtractorInterface):
+class AxonaRecordingInterface(BaseRecordingExtractorInterface):
```

Comment:
Just leaving this here as a demonstration - some of the current interfaces have the word 'Extractor' in the name. While we're breaking back-compatibility with moving things into `neuroconv.datainterfaces`, what would you all think about standardizing the interface names a bit? @h-mayorquin @weiglszonja @bendichter

Reply:
I think that is a good idea to eliminate the redundancy for those which have Extractor in their name

Reply:
I agree with @h-mayorquin.

Reply:
Note that the base class is named `BaseRecordingExtractorInterface`

Reply:
I thought that is intentional, or do you think it is redundant in the base class too @h-mayorquin ?

Reply:
To some degree, we only have RecordingExtractors we don't have Recording by themselves. I am fine with either RecordingInterface or RecordingExtractorInterface but I think we should have consistent names.

Reply:
Ben and I discussed this and we agreed to remove the "Extractor" from all names of the type "<format name>ExtractorInterface" but keep it in all the Base<extractor type>ExtractorInterfaces to indicate that they must specify an Extractor class of some type from either SpikeInterface or ROIExtractors.

I'll also be making a single general BaseExtractorClass to contain the lazy import logic to reduce duplicated code.